src/main/java/com/bullhornsdk/data/api/BullhornRestCredentials.java

```diff
@@ -2,7 +2,12 @@
 
 public class BullhornRestCredentials {
 
-	private String username;
+    //set restUrl and bhRestToken to "default" for validation of credential-less login
+    private String restUrl = "default";
```

Comment:
Would be better to allow values to default to null, and then perform a null check later. Use of string literals is discouraged.

Reply:
Per your and johnsully83's feedback, I switched this to StringUtils.isNotBlank.